app/src/test/java/uk/ac/ebi/atlas/search/AutocompleteControllerWIT.java

```diff
@@ -73,57 +73,76 @@ void setUp() {
-                .andExpect(jsonPath("$[*].options.label", everyItem(not(empty()))))
-                .andExpect(jsonPath("$[*].options.value", everyItem(not(empty()))));
+        this.mockMvc.perform(get("/json/suggestions/gene_ids")
+          .param("species", "Homo sapiens")
+          .param("query", "asp"))
+          .andExpect(status().isOk())
+          .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8))
+          .andExpect(jsonPath("$[*].label", hasSize(greaterThanOrEqualTo(1))))
+          .andExpect(jsonPath("$[*].options.label", everyItem(not(empty()))))
+          .andExpect(jsonPath("$[*].options.value", everyItem(not(empty()))));
     }
 
     @Test
     void atLeastHumanAndMouseAreSuggested() throws Exception {
-        this.mockMvc
-                .perform(get("/json/suggestions/species"))
-                .andExpect(status().isOk())
-                .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8))
-                .andExpect(jsonPath("$.allSpecies", hasItems("Homo sapiens", "Mus musculus")));
+        this.mockMvc.perform(get("/json/suggestions/species"))
+          .andExpect(status().isOk())
+          .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8))
+          .andExpect(jsonPath("$.allSpecies", hasItems("Homo sapiens", "Mus musculus")));
     }
 
     @Test
     void hasAsManyTopSpeciesAsSpecified() throws Exception {
-        this.mockMvc
-                .perform(get("/json/suggestions/species"))
-                .andExpect(status().isOk())
-                .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8))
-                .andExpect(jsonPath("$.topSpecies", hasSize(AutocompleteController.FEATURED_SPECIES)));
+        this.mockMvc.perform(get("/json/suggestions/species"))
+          .andExpect(status().isOk())
+          .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8))
+          .andExpect(jsonPath("$.topSpecies", hasSize(AutocompleteController.FEATURED_SPECIES)));
     }
 
     @Test
     void canSeeSuggestionsForTheOrganismOrOrganismPartOrDiseaseAndCellType() throws Exception {
-        this.mockMvc
-                .perform(get("/json/suggestions/meta_data")
-                        .param("species", "").param("query", "skin"))
-                .andExpect(status().isOk())
-                .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8));
+        this.mockMvc.perform(get("/json/suggestions/meta_data")
+          .param("species", "")
+          .param("query", "skin"))
+          .andExpect(status().isOk())
+          .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8));
+    }
+
+    @Test
+    void canSeeSuggestionsForTheGeneId() throws Exception {
+        this.mockMvc.perform(get("/json/suggestions/gene_search")
+          .param("species", "")
+          .param("query", "asp"))
+          .andExpect(status().isOk()).andExpect(content()
```

Comment:
Please check the formatting settings of your IDE. Formatting correctly enhances readability of code.

<img width="945" alt="Screenshot 2022-04-08 at 16 32 53" src="https://user-images.githubusercontent.com/13108541/162474724-dfc8fee7-9e48-49b1-ad54-c6389774a3df.png">

Reply:
Sure, It's a big puzzle for me, frequently I'm facing this issue. Let me look at IDE settings.

Reply:
Done - code reformat.